rest-server/bin/common.sh

```diff
@@ -13,22 +13,34 @@ export CYAN='\033[0;36m'         # Cyan--hints and info messages
 export WHITE='\033[0;37m'        # White
 
 
+## BSD/MacOS compatible readlink -f
```

Comment:
@pferrel Here's the possible resolution for BSD/MacOS systems it resembles `readlink -f` behavior to locate HARNESS_HOME

Reply:
Yes, this works! Thanks.

I found another way to replace many foundational commands with GNU versions but this seems somewhat dangerous since it is global and may break macOS specific scripts. I can also replace just `readlink` which seems like maybe a better idea since the BSD/masOS readlink is notoriously problematic.

Reply:
Technically anyone can install GNU coreutils and on mac will have greadlink binary, but we can not rely on this.